accepted/future-releases/language-versioning/feature-specification.md

```diff
@@ -38,80 +38,69 @@ Then the Dart language tools must be able to handle libraries written for differ
+The syntax for choosing a different language version than the default is line consisting only of a single-line comment of the form `// @dart = 2.0` (exactly two slashes, the string `@dart`, a `=`, and a version string which is two decimal numerals separated by `.`) in the initial part of the file, prior to any language declarations. It can be preceded by a `#!` line or by other comments, but not by a `library` declaration or any other declaration. Whitespace (space and tabs) are allowed everywhere except between the slashes, inside the `@dart` string and inside the version string. The comment must be an actual single-line comment, and not, say, a line of that form embedded in a block comment. Block comments traditionally start lines with a `*` character, but that is purely convention, so it would be possible to have a line containing just  `//@dart=2.3` inside a block comment. That will not count as a language version override marker. If there is more than one such marker, only the first one applies, but tools are encouraged to warn about the extra unused markers.
 
-The syntax for choosing a different language level than the default is line consisting only of a single-line comment of the form `// @dart = 2.0` (exactly two slashes, the string `@dart`, a `=`, and a numeral which is two decimal numerals separated by `.`) in the initial part of the file, prior to any language declarations. It can be preceded by a `#!` line or by other comments, but not by a `library` declaration or any other declaration. Whitespace (space and tabs) are allowed everywhere except between the slashes, inside the `@dart` string and inside the version numeral. The comment must be an actual single-line comment, and not a string of that form embedded in a block comment. Block comments traditionally start lines with a `*` character, but that is purely convention, so it would be possible to have a line containing just  `//@dart=2.3` inside a block comment. That will not count as a language version override marker. If there is more than one such marker, only the first one applies, but tools are encouraged to warn about the extra unused markers.
+As soon as any library in a Pub package requires a new language feature, the package itself must require a sufficient SDK version to support the feature. This also updates the package's default language version to the higher version. In order to not have to migrate every library in the package immediately, the author can mark unmigrated packages as such, putting them back at the language version they are compatible with. For a non-breaking language change, such marking is not necessary since the "unmigrated" libraries are already compatible with the newer language version.
```

Comment:
This paragraph confused me at first, since it comes in the middle of the discussion of the `@dart` marker, and so I read "requires a new language feature" as "has a // @dart = xx.yy" marker where `xx.yy` introduces "new language feature".   Also, I think this paragraph is technically wrong, since we allow a library to opt forward to a later language than the default without changing the min SDK or changing the default library version.  Consider rephrasing this?  Maybe replace "requires a new language feature" with "uses a new language feature without being explicitly marked with a language version override which enables that feature"?

Reply:
I'll try rewriting, and I'll probably have to go back and reconsider what I originally intended here. Probably just something about code *wanting* to use a new published language feature (and when it does so, it then requiring it).